apps/ledger-live-mobile/src/components/SelectableAccountsList.tsx

```diff
@@ -206,11 +281,24 @@ const SelectableAccount = ({
     if (!onAccountNameChange) return;
 
     swipedAccountSubject.next({ row: -1, list: -1 });
-    navigation.navigate(ScreenName.EditAccountName, {
-      onAccountNameChange,
-      account,
-    });
-  }, [account, navigation, onAccountNameChange]);
+    if (llmNetworkBasedAddAccountFlow?.enabled) {
+      (navigation as StackNavigationProp<AccountSettingsNavigatorParamList>).navigate(
```

Comment:
Do we need this type assertion here ? Is there a way to avoid it ? We should avoid using type assertions as much as possible as they can introduce nasty and hard to debug errors